api/type.rb

```diff
@@ -474,6 +460,61 @@ def property_ns_prefix
         'Property'
       ]
     end
+
+    # Responsible for handling the logic involved with a single ResourceRefs.
+    # A ResourceRefs is made up of one-or-more of these Individual ResourceRefs.
```

Comment:
Ugh, now I'm wondering if maybe it should be ResourceRef and ResourceRefList just to make the grammar a bit easier. What do you think?